traffic_monitor/cache/stats_over_http.go

```diff
@@ -72,6 +74,14 @@ func statsOverHTTPParse(cacheName string, data io.Reader, pollCTX interface{}) (
 
 	ctx := pollCTX.(*poller.HTTPPollCtx)
 
+	via := ctx.HTTPHeader.Get(rfc.Via)
+	if via != "" {
+		viaRegexSubmatch := regexp.MustCompile(` ([^.]+)`).FindStringSubmatch(via)
```

Comment:
Same here, the compiled regex should be reused, rather than calling `regexp.MustCompile()` each time.